tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/discord/DiscordWebHookProcessor.java

```diff
@@ -84,6 +86,16 @@ public boolean sendDiscordWebHook(String webHookURL, String discordWebHookPayloa
             httpPost.addHeader("Accept-Language", "en-US,en;q=0.5");
             httpPost.addHeader("Content-Type", "application/json");
             httpPost.setEntity(new StringEntity(discordWebHookPayload, HTTP_CHARSET));
+            String httpProxyHost = System.getProperty("http.proxyHost");
+            String httpProxyPort = System.getProperty("http.proxyPort");
+            if (httpProxyHost!=null && httpProxyPort.trim().length()>0 && httpProxyPort!=null) {
```

Comment:
Please insert some spaces to make the code more readable
```suggestion
            if (httpProxyHost != null && httpProxyPort.trim().length() > 0 && httpProxyPort != null) {
```